cves/kernel/CVE-2021-4154.yml

```diff
@@ -452,7 +460,10 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+    In this case there was simply a lapse where the input's type was never verified. 
+    Looking at the commit where this was first introduced it appers that it was made and 
+    then signed off on by the same person. A second pair of eyes might have spotted the issue. 
```

Comment:
Second pair of eyes is an excellent point. This is a good tip that developers can take away from this vulnerability, so many small mistakes are discovered when talking through your code with someone else